gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2139,6 +2155,8 @@ func preprocess1(store Store, ctx BlockNode, n Node) Node {
 						}
 					}
 				case CONTINUE:
+					notAllowedFunc("continue")
```

Comment:
should this logic be merged into `findContinuableNode` and `findBranchLabel`, seems what we need to do is to make these two func correct.

Reply:
You mean `findBreakableNode`, not `findBranchLabel`.